x/milestone/abci/abci.go

```diff
@@ -476,3 +479,7 @@ func ValidateMilestoneProposition(ctx sdk.Context, milestoneKeeper *keeper.Keepe
 
 	return nil
 }
+
+func ShouldErrorOnValidatorNotFound(ctx sdk.Context) bool {
```

Comment:
Very minor: as you only use `ctx.BlockHeight()`, consider passing the height directly to the function as a parameter, rather than the whole context.